### ISSUE TITLE
Problems when running with Java 11 - java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter

### ISSUE THREAD

**Issue by AlanTakLee** [nymanjens/facto#8]
Looks like there's a problem running the binaries after the setup with Java 11.
Ubuntu Server 22, fresh `default-jre` installation too.

Tried downloading binaries and had the conf set up correctly but once I tried getting past the login page, I get hit with 
```[error] a.a.ActorSystemImpl - Uncaught error from thread [application-akka.actor.default-dispatcher-3]: javax/xml/bind/DatatypeConverter, shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for ActorSystem[application]
java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
        at io.jsonwebtoken.impl.Base64Codec.decode(Base64Codec.java:26)
        at io.jsonwebtoken.impl.DefaultJwtBuilder.signWith(DefaultJwtBuilder.java:106)
        at play.api.mvc.JWTCookieDataCodec$JWTFormatter.format(Cookie.scala:719)
        at play.api.mvc.JWTCookieDataCodec.encode(Cookie.scala:578)
        at play.api.mvc.JWTCookieDataCodec.encode$(Cookie.scala:576)
        at play.api.mvc.DefaultJWTCookieDataCodec.encode(Cookie.scala:769)
        at play.api.mvc.FallbackCookieDataCodec.encode(Cookie.scala:743)
        at play.api.mvc.FallbackCookieDataCodec.encode$(Cookie.scala:742)
        at play.api.mvc.DefaultSessionCookieBaker.encode(Session.scala:95)
        at play.api.mvc.CookieBaker.encodeAsCookie(Cookie.scala:415)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at io.jsonwebtoken.impl.Base64Codec.decode(Base64Codec.java:26)
        at io.jsonwebtoken.impl.DefaultJwtBuilder.signWith(DefaultJwtBuilder.java:106)
        at play.api.mvc.JWTCookieDataCodec$JWTFormatter.format(Cookie.scala:719)
        at play.api.mvc.JWTCookieDataCodec.encode(Cookie.scala:578)
        at play.api.mvc.JWTCookieDataCodec.encode$(Cookie.scala:576)
        at play.api.mvc.DefaultJWTCookieDataCodec.encode(Cookie.scala:769)
        at play.api.mvc.FallbackCookieDataCodec.encode(Cookie.scala:743)
```

Did some digging around and the fix could be as easy as adding
`libraryDependencies += "javax.xml.bind" % "jaxb-api" % "2.3.0"`

to the sbt

**Comment by nymanjens** [comment]
I haven't seen this error myself yet, so it's a bit hard to debug without being able to reproduce it.

I released a new version (v3.59) with the `jaxb-api` you requested. Can you check whether it works now?

**Comment by AlanTakLee** [comment]
Yep! works and I can login now thank you!
